src/index.js

```diff
@@ -136,11 +146,13 @@ const setLang = (lng) => {
       if (document.getElementById("travel-restrictions")) {
         drawTravelRestrictions(ddb);
       }
-      prefectureTrajectoryChart = drawPrefectureTrajectoryChart(
-        ddb.prefectures,
-        prefectureTrajectoryChart,
-        LANG
-      );
+      if (ddb.isLoaded()) {
```

Comment:
This is a guard to prevent the trajectory chart from drawing when we first load the page and `i18next` gets run for the first time. This generally runs before data is loaded, so it gets skipped.